lib/Controllers/create.js

```diff
@@ -30,6 +57,9 @@ Create.prototype.write = function(req, res, context) {
         res.header('Location', location);
       }
 
+      // Reload for that all field are present
+      return instance.reload();
```

Comment:
Well first of all, this has worked fine the way it is for a number of users, without reloading the data.

If anything, that should be a top level option to force reloads, because you might not care about using that data after create. Flexibility is key here, and some users might not want to pay the price of two SQL statements per create call if all they want to know is whether the operation succeeded or not.